### ISSUE TITLE
cc-badge: introduce css custom properties

### ISSUE THREAD

**Issue by Galimede** [CleverCloud/clever-components#988]
As we added accessibility to the `cc-product-list` component that uses some `cc-badge`, we found that it lacked some css custom properties. Especially to style the `cc-badge` on focus or on hover.

To be able to better handle this case in the future, we'll need these properties:
* cc-badge-color-border
* cc-badge-color-text
* cc-badge-color-bg
